tests/flow/test_ts_madd.py

```diff
@@ -9,6 +9,7 @@
 def test_madd():
     sample_len = 1024
     Env().skipOnCluster()
+    skip_on_rlec()
```

Comment:
@danni-m Why this test isn't working on RLEC?

Reply:
I don't see a lot of value of testing this on a clustered db, the standalone Redis is good enough for testing this command